src/pyhf/exceptions/__init__.py

```diff
@@ -33,8 +34,8 @@ def __init__(self, ValidationError):
                 self.path += '.{}'.format(item)
         self.path = self.path.lstrip('.')
         self.instance = ValidationError.instance
-        message = '{0}.\n\tPath: {1}\n\tInstance: {2}'.format(
-            ValidationError.message, self.path, self.instance
+        message = '{0}.\n\tPath: {1}\n\tInstance: {2} Schema: {3}'.format(
+            ValidationError.message, self.path, self.instance, self.schema
```

Comment:
It would be great if these were f-strings instead
```suggestion
        message = f"{ValidationError.message}.\n\tPath: {self.path}\n\tInstance: {self.instance} Schema: {self.schema}"
```